### PR TITLE
Update data_mapping.mdx

### PR DESCRIPTION
**Pull request by danielwest-statsig**
remove `

before:
image

after:
image

### CODE REVIEW

**Comment by jasonwzm** [comment]
🚀 Deployed on https://673f897b07ba3167dab2b424--cozy-fox-0defec.netlify.app

File: docs/data-warehouse-ingestion/data_mapping.mdx [hunk 1]
```diff
@@ -30,7 +30,7 @@ Events that are emitted by your application to measure the ongoing impact of you
 
 
 #### Optional
-`
```
Comment (jairo-statsig) [inline docs/data-warehouse-ingestion/data_mapping.mdx:33]: What was this doing before hand?

**Review by tore-statsig — APPROVED** [review]
+3

File: docs/data-warehouse-ingestion/data_mapping.mdx (hunk 1 above)
Comment (jairo-statsig) [reply docs/data-warehouse-ingestion/data_mapping.mdx:33]: Oh wow that's a good catch!

Comment (jairo-statsig) [reply docs/data-warehouse-ingestion/data_mapping.mdx:33]: +3

**Review by jairo-statsig — APPROVED** [review]

**Pull request merged**